File: app/src/main/java/com/example/billeteramodulo4/Send_Money.java
package com.example.billeteramodulo4;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

import androidx.appcompat.app.AppCompatActivity;

public class Send_Money extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.send_money);

        // Abrir nueva activity

        ImageButton boton = findViewById(R.id.sendhome);

        boton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                //ir a activity 4
                Intent i = new Intent(Send_Money.this, Profile_Page.class);
                startActivity(i);
            }
        });


    }
}

File: app/src/main/java/com/example/billeteramodulo4/SendMoney.kt
package com.example.billeteramodulo4

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.navigation.fragment.findNavController
import com.example.billeteramodulo4.databinding.FragmentSendMoneyBinding


class SendMoney : Fragment() {

    private lateinit var binding: FragmentSendMoneyBinding


    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {

        binding = FragmentSendMoneyBinding.inflate(inflater, container, false)
        val root = binding.root

        binding.btnenviadinerosendpage.setOnClickListener {
            findNavController().navigate(R.id.action_sendMoney_to_homePage)
        }

        return root
    }

}